core/include/signed_hdr.h

```diff
@@ -57,6 +58,38 @@ struct shdr_bootstrap_ta {
 	uint32_t ta_version;
 };
 
+/**
```

Comment:
Everything else in this file uses `shdr_` or `SHDR_` prefix. Shouldn't these do that too?

Reply:
Ok, will use these prefixes.